.ci/scripts/list-s3-artifacts.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
This will only help with detecting whether the vault secrets are broken before even running the release.